src/js/structs/List.js

```diff
@@ -53,6 +53,30 @@ module.exports = class List {
     this.list.push(cast.call(this, item));
   }
 
+  /**
+   * @param {*} items
+   * @returns {List} List (or child class) containing concatenated items
+   */
+  concatItems(...items) {
+
+    // Prepend current list
+    items.unshift(this);
+
+    // Get items from lists
+    items = items.map(function (item) {
+      if ((item instanceof List)) {
+        return item.getItems();
+      }
+
+      return item
```

Comment:
I don't do this to offend you, but there is no eslint rule to enforce the semicolon and my IDE is not adding them as they aren't required by the ECMAScript (Automatic Semicolon Insertion).